google/ggrc-core
# GGRC-4169 Fix footer styles

# Issue description

Page content is overlapped by footer + the page has 2 scrolls.

# Steps to test the changes

1. Login as Admin user 
2. Go to administration page and add GCAs to Regulation (at least 10 GCAs) 
3. Create Regulation and open it.
4. Scroll down to see "Created date" and "Last update by" 
Expected Result: "Created date" and "Last update by" should be shown in the bottom of the page if object contains GCAs

# Solution description

The cause of the issue - the time of resize/load callback execution. Sometimes navbar is rendered after these 2 callbacks are called. Which means that object-area height is calculated as winHeight - $topNav.height() [equals 0] - other elements height => it's 40 px (topNavHeight) bigger that it should be. I examined resize_area function and came to conclusion that it's possible to have the same page layout without js calculation of elements height. So I updated styles, layout to consume flex.

# Sanity checklist




# PR Review checklist
